frontend/src/App.css

```diff
@@ -156,11 +167,57 @@ main {
   margin: 1rem;
 }
 .bosslife {
+  font-family: "Bangers";
+
   display: flex;
   flex-direction: column;
   align-items: center;
 }
-
+.chactersfight {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.progressgame {
+  position: relative;
+  margin-bottom: 0.5rem;
+}
+.progressgame:after {
+  content: attr(data-label);
+  position: absolute;
+  top: 0.35rem;
+  left: 26.5vw;
```

Comment:
this relies on the viewport size : this is unpredictable. are you trying to center ?